src/app/workspaces/workspace-details/workspace-details.controller.ts

```diff
@@ -215,12 +216,14 @@ export class WorkspaceDetailsController {
     };
   }
 
-  private setWorkspaceDetails(workspaceDetails: che.IWorkspace): void {
+  private setWorkspaceDetails(workspaceDetails: che.IWorkspace, force: boolean = false): void {
```

Comment:
please add jsdoc and describe what is `force`.

Reply:
done